python/paddle/__init__.py

```diff
@@ -40,7 +40,7 @@
 
 from .framework import disable_static  # noqa: F401
 from .framework import enable_static  # noqa: F401
-from .framework import in_dynamic_mode  # noqa: F401
+from .framework import in_dygraph_mode as in_dynamic_mode  # noqa: F401
```

Comment:
I suggest not having `as` here, rather to modify `from ..fluid.framework import in_dygraph_mode as in_dynamic_mode`  in `python/paddle/framework/__init__.py`. later, after implement in fluid is migrated to the `python/paddle/framework/`, it can ensure that the API name here is the same as the function implementation in the framework.

Reply:
done